homeassistant/components/unifi/switch.py

```diff
@@ -34,19 +36,38 @@
         return
 
     switches = {}
+    switches_off = []
+
+    registry = await entity_registry.async_get_registry(hass)
+
+    # Restore clients that is not a part of active clients list.
+    for entity in registry.entities.values():
+
+        if entity.config_entry_id == config_entry.entry_id and \
+                entity.unique_id.startswith('poe-'):
+
+            _, mac = entity.unique_id.split('-', 1)
+
+            if mac in controller.api.clients or \
+                    mac not in controller.api.clients_all:
+                continue
+
+            client = controller.api.clients_all[mac]
+            controller.api.clients.process_raw([client.raw])
+            switches_off.append(entity.unique_id)
 
     @callback
     def update_controller():
         """Update the values of the controller."""
-        update_items(controller, async_add_entities, switches)
+        update_items(controller, async_add_entities, switches, switches_off)
 
     async_dispatcher_connect(hass, controller.event_update, update_controller)
 
     update_controller()
 
 
 @callback
-def update_items(controller, async_add_entities, switches):
+def update_items(controller, async_add_entities, switches, switches_off=[]):
```

Comment:
We shouldn't use mutable objects as default arguments.

Reply:
Damn it! You're right. Will fix